README.md

```diff
@@ -69,18 +69,12 @@ The dispatcher should call `emqx_cm:is_channel_connected/1` to check if the subs
 
 If there is already a message in flight, the `maybe_send` notification will be ignored.
```

Comment:
If the inflight message is not the same as latest one, we rely on the next hartbeat to trigger the sending?

Reply:
yes. only one message inflight for flow control since SDV payload tend to be large.